README.md

```diff
@@ -309,160 +192,6 @@ curl helloenv.stage.com
-
-![PR created using workflow in helloenv-app repo](/assets/img/Blog/automatic-image-update-to-git-using-flux-github-actions/pr-created-using-flux-helloenv-app-repo.png)
-
-curl to the prod DNS should give you the latest tag as the version. 
-
-```sh
-$ curl helloenv.prod.com
-
-This is production environment (Version: 1.0.2)
-```
-
-## Incident Management
-
-During an incident, you may want to halt Flux from updating images in your Git repository. You can accomplish this by suspending image automation either in-cluster or by editing the ImageUpdateAutomation manifest in Git.
-
-### In-cluster suspension
-
-You can suspend image automation directly in a cluster using the following command:
-
-```sh
-flux suspend image update helloenv-prod
-```
-
-Alternatively, you can suspend image automation by editing the ImageUpdateAutomation manifest in Git. Here's an example of the manifest:
-
-```yaml
-kind: ImageUpdateAutomation
-metadata:
-  name: helloenv-prod
-  namespace: flux-system
-spec:
-  suspend: true
-```
-
-### Resuming automation
-
-Once the incident is resolved, you can resume the automation using the following command:
-
-```sh
-flux resume image update helloenv-prod
-```
-
-### Pausing automation for a specific image
-
-If you want to pause automation for a particular image only, you can suspend and resume image scanning for that specific image. For example:
-
-```sh
-flux suspend image repository helloenv
-```
-
-### Reverting image updates
-
-Assuming you've configured Flux to update an application to its latest stable version and an incident occurs, you can instruct Flux to revert to a previous image version.  
-
-#### Reverting via command
-
-For instance, to revert from version 1.0.1 to 1.0.0, you can use the following command:
-
-```sh
-flux create image policy helloenv-prod --image-ref=helloenv --select-semver=1.0.0
-```
-
-#### Reverting via Git manifest
-
-You can also make this change by editing the ImagePolicy manifest in Git. Here's an example of the manifest:
-
-```yaml
-kind: ImagePolicy
-metadata:
-  name: helloenv-prod
-  namespace: flux-system
-spec:
-  policy:
-    semver:
-      range: 1.0.0
-```
-
-### Updating the image policy
-
-When a new version, e.g., 1.0.2, becomes available, you can update the policy again to consider only versions greater than 1.0.1. This can be achieved using the following command:
-
-```sh
-flux create image policy helloenv-prod --image-ref=helloenv --select-semver=">1.0.1"
-```
+## What's next?
 
-This change will prompt Flux to update the podinfo deployment manifest in Git and roll out the specified image version in-cluster.
+Refer to the [Automatic Image Update to Git using Flux and GitHub Actions Blog](https://infracloud.io/blogs/automatic-image-update-to-git-using-flux-github-actions/) for next steps, detailed explanations and incident management.
```

Comment:
Let's move this section to top? We can say the post has a detailed walk through with explanation. That way if someone lands on this repo they can go and read the post if they want to.